transport/pool.go

```diff
@@ -35,6 +46,39 @@ func NewConnPool(addr string, cfg ConnConfig) (*ConnPool, error) {
 
 const defaultPort = 19042 // TODO: change to 9042 after fixing the scylla supported bug.
 
+func (p *ConnPool) UseKeyspace(keyspace string) error {
+	if err := validKeyspace(keyspace); err != nil {
+		return err
+	}
+	resp := make(keyspaceRespCh)
+	p.keyspaceReqCh <- keyspaceReq{
+		keyspace: keyspace,
+		resp:     resp,
+	}
+	return <-resp
+}
+
+func validKeyspace(keyspace string) error {
```

Comment:
s/valid/validate/ 

Valid more isValid would return bool, typically we call that validateX